content/docs/user-guide/contributing/docs.md

```diff
@@ -81,36 +77,60 @@ Launch the server locally with:
 $ yarn develop
 ```
 
-This will start the server on the default port, `3000`. Visit
-`http://localhost:3000/` and navigate to the docs in question. This will also
-enable the Git pre-commit hook that will be formatting your code and
+This will start the server on the default port, `8000`. Visit
+`http://localhost:8000/` and navigate to the page in question. This will also
+enable the Git pre-commit hook that will be formatting and linting your code and
 documentation files automatically.
 
-### Debugging
+### All available commands
+
+To build the project and run it:
+
+- `yarn build` - build assets in the `public` directory (`gatsby build`).
+- `yarn develop` - run development server with hot reload (`gatsby develop`).
```

Comment:
Put develop before build? Not sure we need to specify the underlying gatsby command.

Reply:
build should probably still go first, removed gatsby thingy

Reply:
To my ming you use develop first, when you're happy you do build + serve. But OK not a big deal obviously.

Reply:
k, let's change it ... not a big deal